### ISSUE TITLE
RPC vector layer cleanup

### ISSUE THREAD

**Issue by ZakarFin** [oskariorg/oskari-docs#214]
Currently you can add/remove vector features to the map via RPC. However there is no way to remove layers from the map once you are done with them. You can get close by removing all features from a layer but if the application is meant to be used for extended period of time without refreshing and layers are constantly added and removed these empty vector layers might pile up and affect performance.

We should add a way to remove an unwanted vector layer when the software wants to clean up.

**Comment by ZakarFin** [comment]
We are planning on adding a remove-param in the options of VectorLayerRequest. We are planning on keeping the layerId as required parameter for removal. If we would allow removal of all layers it's very easy to accidentally break something like removing the thematic map layer or marker layer. Let me know if you guys think we should allow full removal even with the risk of things breaking accidentally by calling it.

**Comment by ZakarFin** [comment]
This will be included in 2.1.0 (https://github.com/oskariorg/oskari-frontend/pull/1351):
```
Oskari.getSandbox().postRequestByName('VectorLayerRequest', [{
    layerId: 'MY_VECTOR_LAYER', // existing id,
    remove: true
}]); 
```

**Comment by ZakarFin** [comment]
Feature added in 2.1.0
